config/routes.rb

```diff
@@ -36,4 +40,10 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/applications/new", to: "applications#new"
+  post "/applications", to: "applications#create"
+  get "/applications/:id", to: "applications#show"
+  post "/applications/:id", to: "pet_applications#create"
```

Comment:
I think to make this even more RESTful you could add
```ruby
post "/applications/:id/pets", to: "pet_applications#create"
```
This way you are creating the association between an application and a pet.